templates/macros/_vf-latest-news.jinja

```diff
@@ -0,0 +1,95 @@
+{#
+  Params:
+    title_text(string) (required): The title of the section, which also links to the main blog page.
+    tag_name (string) (required): The URL-friendly slug of a tag to filter the articles by.
+    tag_id (string) (required): The ID of a tag to filter the articles by.
+#}
+
+{% macro vf_latest_news(
+  title_text,
+  tag_name="",
+  tag_id=""
+  ) -%}
+
+  {% set heading_url = "/blog" %}
+  {% if tag_name %}
+    {% set heading_url = "/blog/tag/" + tag_name %}
+  {% endif %}
+
+  {% set gtm_event_label = "ubuntu.com" %}
+  {% if tag_name %}
+    {% set gtm_event_label = gtm_event_label + "/" + tag_name %}
+  {% else %}
+    {% set gtm_event_label = gtm_event_label + " homepage" %}
+  {% endif %}
+
+  <noscript>
+    <section class="p-strip">
+      <div class="row">
+        <h3>
+          <a href="{{ heading_url }}">{{ title_text }}</a>
+        </h3>
+      </div>
+    </section>
+  </noscript>
+
+  <section class="u-hide p-section--deep" data-js="latest-news">
+    <hr class="p-rule is-fixed-width" />
+    <div class="row p-section--shallow">
+      <h2>
+        <a href="{{ heading_url }}"
+           onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'blog', 'eventAction' : 'clicks blog feed link', 'eventLabel' : '{{ gtm_event_label }}', 'eventValue' : undefined });">{{ title_text }}</a>
+      </h2>
+    </div>
+    <div class="row">
+      <div class="col-12">
+        <div id="latest-articles-container" class="p-equal-height-row--wrap">
+          <div style="min-height: 9.1rem">
+            <i class="p-icon--spinner u-animation--spin">Loading...</i>
+          </div>
+        </div>
+      </div>
+    </div>
+
```

Comment:
![image](https://github.com/user-attachments/assets/f27b36ae-3ef3-4011-bd42-aede44a5672d)
Are these lines on medium screen part of design?